specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/WebApps.json

```diff
@@ -262,6 +262,9 @@
           "204": {
             "description": "Successfully deleted web app."
           },
+          "202": {
+            "description": "Asynchronous operation in progress."
```

Comment:
Add "Location" header also in the 202 response.

Reply:
I don't see headers added anywhere else for 202 responses, can you please share the sample if you have any.

Reply:
Here's an example: [https://github.com/Azure/azure-rest-api-specs/blob/c7b50cac59a57b2c90f2acdb66ed9252825ed232/specification/network/resource-manager/Microsoft.Network/preview/2023-08-01-preview/networkSecurityPerimeter.json#L713](https://github.com/Azure/azure-rest-api-specs/blob/c7b50cac59a57b2c90f2acdb66ed9252825ed232/specification/network/resource-manager/Microsoft.Network/preview/2023-08-01-preview/networkSecurityPerimeter.json#L713).

Reply:
Thanks for sharing the sample, I have added the headers.